src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2711,6 +2676,45 @@ codet java_bytecode_convert_methodt::convert_instructions(
   return code;
 }
 
+void java_bytecode_convert_methodt::draw_edges_from_ret_to_jsr(
+  java_bytecode_convert_methodt::address_mapt &address_map,
+  std::vector<unsigned int> &jsr_ret_targets,
+  const std::vector<
+    std::vector<java_bytecode_parse_treet::instructiont>::const_iterator>
```

Comment:
Please introduce a typedef, this is very hard to parse for humans, and the same type is used multiple times.